.gitlab-ci.yml

```diff
@@ -978,10 +983,13 @@ test_full_integration:
     # only do automatic test suite selection if the user wasn't specific
     # run.sh will pick up the SPECIFIC_INTEGRATION_TEST var
     - if [ -z "$SPECIFIC_INTEGRATION_TEST" ]; then
-      case $INTEGRATION_TEST_SUITE in
-      "enterprise") export SPECIFIC_INTEGRATION_TEST="Enterprise";;
-      "open") export SPECIFIC_INTEGRATION_TEST="not Enterprise";;
-      esac fi
+        case $INTEGRATION_TEST_SUITE in
+          "enterprise")
+            export SPECIFIC_INTEGRATION_TEST="Enterprise";;
+          "open")
+            export SPECIFIC_INTEGRATION_TEST="not Enterprise";;
+        esac
+      fi
```

Comment:
All this time I thought it was because Gitlab didn't understand indented yaml lines! :laughing:

Reply:
That was my original assumption. It took months to realize it actually just ignores all space (that is why we must always end statements with `; `.